src/server/game/Spells/Spell.h

```diff
@@ -547,6 +547,7 @@ class TC_GAME_API Spell
         ObjectGuid m_originalCastId;
         bool m_fromClient;
         uint32 m_castFlagsEx;
+        uint32 m_delay;
```

Comment:
Always initialize all fields in the constructor

Reply:
Done, I also passed m_delay to be 1ms by default.